sympy/geometry/tests/test_polygon.py

```diff
@@ -98,7 +103,8 @@ def test_polygon():
         Polygon(Point(10, 10), Point(14, 14), Point(10, 14))) == 6 * sqrt(2)
     assert p5.distance(
         Polygon(Point(1, 8), Point(5, 8), Point(8, 12), Point(1, 12))) == 4
-    with warns(UserWarning, match="Polygons may intersect producing erroneous output"):
+    with warns(UserWarning, 
```

Comment:
It seems like you (probably) pressed enter after the space, and, hence, there is a trailing space on this line:

`AssertionError: File contains trailing whitespace: /home/travis/build/sympy/sympy/sympy/geometry/tests/test_polygon.py, line 106.`